tanzu-java-web-app/accelerator.axl

```diff
@@ -0,0 +1,76 @@
+engine {
+
+  // Include files for the selected build tool
+  if (#buildTool == "maven") {
+    Exclude({"*gradle*", "Tiltfile_gradle_vscode", "Tiltfile_gradle_intellij"})
+    + InvokeFragment(reference: "build-wrapper-maven")
+  }
+  if (#buildTool == "gradle") {
+    applyTo("settings.gradle") {
+      ReplaceText({{text: "tanzu-java-web-app", with: #projectName}})  
+    }
+    if (#ide == "intellij") {
+      Exclude({"Tiltfile", "Tiltfile_gradle_vscode"})
+      RewritePath(regex: "Tiltfile_gradle_intellij", rewriteTo: 'Tiltfile')
+    }
+    if (#ide == "vscode") {
+      Exclude({"Tiltfile", "Tiltfile_gradle_intellij"})
+      RewritePath(regex: "Tiltfile_gradle_vscode", rewriteTo: 'Tiltfile')
+    }
+    {
+      Exclude({"pom.xml"})
+      + InvokeFragment(reference: "build-wrapper-gradle")
```

Comment:
Not an action item for this PR, but if we're going to promote chain over merge going forward, our fragments would be better off touching only the files they care about and leaving the others as-is.
This 4 lines block would then just be
```
  InvokeFragment("build-wrapper-gradle")
```

Reply:
Not sure what we should promote in terms of chain over merge, but it seemed more natural to use a chain with the new DSL syntax. Throwing in a bunch of `+` to get a merge , to me, seems fine on smaller snippets but not at the top level.